python/tests/builder/test_kernel_builder.py

```diff
@@ -1243,6 +1244,16 @@ def test_list_subscript():
     cudaq.sample(kernelAndArgs[0], False, [], [], [])
 
 
+def test_u3_op():
+    kernel = cudaq.make_kernel()
+    q = kernel.qalloc(1)
+    # TODO: Allow use of only `q`
```

Comment:
Does this mean you want python to accept a list of qubits in the final position? I assume that would be checked such that the list must contain exactly one qubit or it is ill-formed.

Reply:
I changed the comment - the current behavior requires `q[0]`, asking if we should / need to support just using `q` and if `q` has multiple qubits, the operation is applied to all

Reply:
Fixed